### ISSUE TITLE
EAS build doesn't work

### ISSUE THREAD

**Issue by ambient-okada** [bndkt/react-native-app-clip#16]
I tried to build the expo managed app which is installed react-native-app-clip following instruction in README with eas build.
But eas build failed in process of run fastlane with following error log.
Is there any solution or workaround?

```
$ xcodebuild -showBuildSettings -workspace ./appclipdemo.xcworkspace -scheme appclipdemo -configuration Release
[stderr]
--- xcodebuild: WARNING: Using the first of multiple matching destinations:
[stderr]
{ platform:macOS, arch:arm64, variant:Designed for [iPad,iPhone], id:0000FE00-66D45333AEB93EDA }
[stderr]
{ platform:iOS, id:dvtdevice-DVTiPhonePlaceholder-iphoneos:placeholder, name:Any iOS Device }
[stderr]
{ platform:iOS Simulator, id:dvtdevice-DVTiOSDeviceSimulatorPlaceholder-iphonesimulator:placeholder, name:Any iOS Simulator Device }
[stderr]
{ platform:iOS Simulator, id:94EE5B62-6A73-4410-8C64-A56FBFD9C55D, OS:16.2, name:iPad (10th generation) }
[stderr]
{ platform:iOS Simulator, id:537FF0C8-8B14-4E2E-9828-0ED246BE9E38, OS:16.2, name:iPad Air (5th generation) }
[stderr]
{ platform:iOS Simulator, id:777A520C-1C1B-484A-99C6-35943FA97067, OS:16.2, name:iPad Pro (11-inch) (4th generation) }
[stderr]
{ platform:iOS Simulator, id:A5447295-3264-4816-9C78-50E5F35C1122, OS:16.2, name:iPad Pro (12.9-inch) (6th generation) }
[stderr]
{ platform:iOS Simulator, id:067F74D9-EEC3-4066-B8A8-B3EAD0EE5649, OS:16.2, name:iPad mini (6th generation) }
[stderr]
{ platform:iOS Simulator, id:3A1FD6BA-A84C-4728-A0AA-E7D9DF147BA7, OS:16.2, name:iPhone 14 }
[stderr]
{ platform:iOS Simulator, id:5D252FD0-2816-4FAE-B85D-C29DC8D2C73C, OS:16.2, name:iPhone 14 Plus }
[stderr]
{ platform:iOS Simulator, id:AA475350-BBB4-4568-A722-4528CF187FA8, OS:16.2, name:iPhone 14 Pro }
[stderr]
{ platform:iOS Simulator, id:01B24393-FC59-4FEC-87FD-338E47BABDAE, OS:16.2, name:iPhone 14 Pro Max }
[stderr]
{ platform:iOS Simulator, id:88B2387A-FF6D-4979-9155-B06611A97148, OS:16.2, name:iPhone SE (3rd generation) }
Command timed out after 3 seconds on try 1 of 4, trying again with a 6 second timeout...
[stderr]
2023-06-12 23:56:25.098 xcodebuild[4310:11586] Requested but did not find extension point with identifier Xcode.InterfaceBuilderBuildSupport.PlatformDefinition
[stderr]
--- xcodebuild: WARNING: Using the first of multiple matching destinations:
[stderr]
{ platform:macOS, arch:arm64, variant:Designed for [iPad,iPhone], id:0000FE00-66D45333AEB93EDA }
[stderr]
{ platform:iOS, id:dvtdevice-DVTiPhonePlaceholder-iphoneos:placeholder, name:Any iOS Device }
[stderr]
{ platform:iOS Simulator, id:dvtdevice-DVTiOSDeviceSimulatorPlaceholder-iphonesimulator:placeholder, name:Any iOS Simulator Device }
[stderr]
{ platform:iOS Simulator, id:94EE5B62-6A73-4410-8C64-A56FBFD9C55D, OS:16.2, name:iPad (10th generation) }
[stderr]
{ platform:iOS Simulator, id:537FF0C8-8B14-4E2E-9828-0ED246BE9E38, OS:16.2, name:iPad Air (5th generation) }
[stderr]
{ platform:iOS Simulator, id:777A520C-1C1B-484A-99C6-35943FA97067, OS:16.2, name:iPad Pro (11-inch) (4th generation) }
[stderr]
{ platform:iOS Simulator, id:A5447295-3264-4816-9C78-50E5F35C1122, OS:16.2, name:iPad Pro (12.9-inch) (6th generation) }
[stderr]
{ platform:iOS Simulator, id:067F74D9-EEC3-4066-B8A8-B3EAD0EE5649, OS:16.2, name:iPad mini (6th generation) }
[stderr]
{ platform:iOS Simulator, id:3A1FD6BA-A84C-4728-A0AA-E7D9DF147BA7, OS:16.2, name:iPhone 14 }
[stderr]
{ platform:iOS Simulator, id:5D252FD0-2816-4FAE-B85D-C29DC8D2C73C, OS:16.2, name:iPhone 14 Plus }
[stderr]
{ platform:iOS Simulator, id:AA475350-BBB4-4568-A722-4528CF187FA8, OS:16.2, name:iPhone 14 Pro }
[stderr]
{ platform:iOS Simulator, id:01B24393-FC59-4FEC-87FD-338E47BABDAE, OS:16.2, name:iPhone 14 Pro Max }
[stderr]
{ platform:iOS Simulator, id:88B2387A-FF6D-4979-9155-B06611A97148, OS:16.2, name:iPhone SE (3rd generation) }
[stderr]
2023-06-12 23:56:26.146 xcodebuild[4320:11658] Requested but did not find extension point with identifier Xcode.InterfaceBuilderBuildSupport.PlatformDefinition
```

**Comment by bndkt** [comment]
According to Google, [this](https://stackoverflow.com/a/73253436) could be the answer. Did you by any chance install Xcode beta?

**Comment by okataku** [comment]
The environment of eas build I use is following.
I'm using the default environment which expo.dev offers.
Do you need any customization for eas build environment?

```
Creating new worker instance
M1, 4 cores, 16 GB RAM
Using VM template "macos-monterey-12.6-xcode-14.2" running macOS Monterey 12.6
Installed software:
- Xcode 14.2 (14C18)
- Node.js 16.18.1
- Yarn 1.22.17
- pnpm 7.27.0
- npm 8.1.2
- fastlane 2.211.0
- CocoaPods 1.11.3
- Ruby 2.7
- Expo CLI 6.0.0
```

**Comment by bndkt** [comment]
I just released a new version (0.2.0). Can you please check if the problem still exists? If yes, please share a MRE (see https://vercel.com/guides/creating-a-minimal-reproducible-example for a good introduction) and reopen this issue.
